src/backend_model.h

```diff
@@ -47,7 +47,7 @@ class TritonModelInstance;
 class TritonModel : public Model {
  public:
   static Status Create(
-      InferenceServer* server, const std::string& model_repository_path,
+      InferenceServer* server, const std::string& original_model_path,
```

Comment:
Just call `model_path`

Reply:
That would cause a conflict with a local variable. https://github.com/triton-inference-server/core/blob/388e9eb970eacff0214496aae72ecd077b316f28/src/backend_model.cc#L107

model_path represents the model path used, whether local or original. It seems like the call to LocalizeDirectory() sets these differently for different filesystems, so I thought keeping the nomenclature original_path made sense. If you'd prefer something else, let me know.

Reply:
I see, then I am okay with that. Or you may change the local variable `model_path` to `localized_model_path`.

Reply:
Updated.